DataFormats/Reconstruction/include/ReconstructionDataFormats/GlobalTrackID.h

```diff
@@ -61,46 +64,97 @@ class GlobalTrackID : public AbstractRef<25, 5, 2>
   };
 
   using AbstractRef<25, 5, 2>::AbstractRef;
+  typedef o2::gpu::gpustd::bitset<NSources> mask_t; // std::bitset<NSources> not supported on GPU
```

Comment:
Well, since I am always storing 32 bit, it should be tolerant to this. But I assume the problem will still be that the template parameter is part of the type, so ROOT would probably again be confused if we change it. So I'll switch to 32 bit here as well.